cmd/snap/cmd_prepare_image.go

```diff
@@ -48,18 +48,20 @@ core device images.
 		func() flags.Commander {
 			return &cmdPrepareImage{}
 		}, map[string]string{
+			// TRANSLATORS: This should not start with a lowercase letter.
 			"extra-snaps": "Extra snaps to be installed",
-			"channel":     "The channel to use",
+			// TRANSLATORS: This should not start with a lowercase letter.
+			"channel": "The channel to use",
```

Comment:
are these two strings passed through i18n elsewhere? if not, then we either need to remove the `TRANSLATORS:` comment or add the `i18n.G` function call